react/lib/grid-stack-context.ts

```diff
@@ -1,35 +0,0 @@
-import type { GridStack, GridStackOptions, GridStackWidget } from "gridstack";
```

Comment:
hey don't delete those files. I actually wanted to start shipping them like I do for /angular 
I started looking at the code and was planning to do this weekend... I wanted to match better what I did for angular wrapper (extending WidgetItem to have similar to 'selector' + 'input' which are angular centric but conceptually the same as name + props you have.